index.php.in

```diff
@@ -5,44 +5,67 @@ header("X-Powered-By: ");
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
     <meta charset="utf-8" />
-    <title>JQuery Terminal Emulator Plugin</title>
+    <title>jQuery Terminal Emulator Plugin</title>
     <link rel="canonical" href="https://terminal.jcubic.pl"/>
     <meta name="author" content="Jakub Jankiewicz - jcubic&#64;jcubic.pl"/>
-    <meta name="Description" content="jQuery plugin for Command Line applications. Automatic JSON-RPC, custom object or a function. History, Authentication, Bash Shortcuts. Tab completion."/>
+    <meta name="Description"
+      content="jQuery plugin for Command Line applications.
+               Automatic JSON-RPC, custom object or a function.
+               History, Authentication, Bash Shortcuts. Tab completion."/>
     <meta property="fb:admins" content="100000949516439" />
-    <link rel="sitemap" type="application/xml" title="Sitemap" href="/sitemap.xml"/>
-    <meta name="keywords" content="jquery,terminal,interpreter,console,bash,history,authentication,ajax,server,client"/>
+    <link rel="sitemap" type="application/xml" title="Sitemap"
+          href="/sitemap.xml"/>
+    <meta name="keywords"
+      content="jquery, terminal, interpreter, console, bash, history,
+      authentication, ajax, server, client"/>
     <link rel="shortcut icon" href="favicon.ico"/>
-    <link rel="alternate" type="application/rss+xml" title="Comments RSS" href="https://terminal.jcubic.pl/comments-rss.php"/>
-    <link rel="stylesheet" href="css/style.css?<?= md5(file_get_contents('css/style.php')) ?>"/>
-    <link href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono" rel="stylesheet" type="text/css"/>
+    <link rel="alternate" type="application/rss+xml" title="Comments RSS"
+      href="https://terminal.jcubic.pl/comments-rss.php"/>
+    <link rel="stylesheet"
+      href="css/style.css?<?= md5(file_get_contents('css/style.php')) ?>"/>
+    <link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono" />
     <link href="css/jquery-ui-1.8.7.custom.css" rel="stylesheet"/>
-    <link href="css/jquery.terminal.min.css?<?= md5(file_get_contents('css/jquery.terminal.min.css')) ?>" rel="stylesheet"/>
+    <?php $css_0 = "css/jquery.terminal.min.css"
+          $css_1 = md5(file_get_contents($css_0))
+          $css_2 = $css_0 . "?" . $css_1 ?>
+    <link href="<?php echo $css_2 ?>" rel="stylesheet"/>
```

Comment:
what's wrong with link in single line, it's source code it's not displayed to the user. Also I don't like this global variables, and concatenations.